### PR TITLE
Automatically create GitHub releases

### LINKED ISSUE #57: Automatically create GitHub releases
Use the `gh` CLI to create a new GitHub release as part of the existing release flow in our CI code. GitHub releases should include:
* a compressed archive for each of our supported platforms, 
* release notes, 
* and a list of contributions/commits since last release.

### PR DESCRIPTION
**Pull request by matthew-levan**
Resolves #57.

### CODE REVIEW

**Comment by matthew-levan** [comment]
Is this still necessary? If not, please close it.

**Comment by sethfork** [comment]
A common pattern of behavior is checking releases on `urbit/urbit`. It now seems `vere` is the place to go, which is fine - it would just be good to continue actually doing releases. If there's no technical reason to block this, it should be accepted.

File: .github/workflows/shared.yml [hunk 1]
```diff
@@ -161,7 +166,26 @@ jobs:
...
+      
+      - name: Create a GitHub release
+        if: ${{ inputs.release }}
+        run: |
+          # Only create the GitHub release on `linux-x86_64`.
+          # Upload the other assets from the other platforms' jobs.
+          version="v$(cat ./pkg/vere/VERSION)"
+          if ${{ matrix.target == 'linux-x86_64' }}; then
+            # Create a new tag.
+            git tag urbit-v$version
```
Comment (sethfork) [inline .github/workflows/shared.yml:178]: Rename tag from `urbit-v$version` to `vere-v$version`

File: .github/workflows/shared.yml [hunk 2]
```diff
@@ -161,7 +166,26 @@ jobs:
...
+            git push origin --tags
+            # Create a draft release with automatically-generated notes and
+            # source code archives.
+            gh release create urbit-v$version \
```
Comment (sethfork) [inline .github/workflows/shared.yml:182]: Use `vere-v$version` for GH release

File: .github/workflows/shared.yml [hunk 3]
```diff
@@ -161,7 +166,26 @@ jobs:
...
+              --draft \
+              --generate-notes
+          fi
+          # Upload the binary to the release.
+          tar -czf ${{ matrix.target }}.tgz ${{ env.urbit_static }}
+          gh release upload urbit-v$version ${{ matrix.target }}.tgz
```
Comment (sethfork) [inline .github/workflows/shared.yml:188]: Likewise, switch `urbit` to `vere` here

**Review by sethfork — CHANGES_REQUESTED** [review]

**Comment by matthew-levan** [comment]
I have some updates here but I can't push to my original feature branch since being removed from the `urbit` organization on GH. I'll push once I get re-added, or just fork the repo if it takes too long.

**Comment by matthew-levan** [comment]
Closing in favor of #136.

**Pull request closed without merging**